editor/src/core/property-controls/property-controls-local.ts

```diff
@@ -106,6 +106,7 @@ async function componentDescriptorForComponentToRegister(
   const parsedVariantsUnsequenced = await Promise.all(parsedInsertOptionPromises)
   const parsedVariants = sequenceEither(parsedVariantsUnsequenced)
 
+  // TODO
```

Comment:
TODO what?

Reply:
forgot to push the commit that removes it https://github.com/concrete-utopia/utopia/pull/5087/commits/d0ae89e2a65ba204f048cb70e0cbc8e5ccac7c17